cmd/deploy/deploy.go

```diff
@@ -287,6 +283,17 @@ func (dc *DeployCommand) RunDeploy(ctx context.Context, deployOptions *Options)
 		oktetoLog.Infof("could not create temporal kubeconfig %s", err)
 		return err
 	}
+
+	_, proxyConfig, err := dc.K8sClientProvider.Provide(kconfig.Get([]string{dc.TempKubeconfigFile}))
```

Comment:
I think you can do `dc.Kubeconfig.Read` to get the cfg, that way we don't have to open and read a file